src/test/ui/panics/panic-short-backtrace-windows-x86_64.run.stderr

```diff
@@ -5,5 +5,4 @@ stack backtrace:
    2: c
    3: b
    4: a
-   5: some Rust fn
```

Comment:
This is just a guess. 🤷‍♀️

Reply:
> 💔 Test failed - [checks-actions](https://github.com/rust-lang-ci/rust/runs/4347386377?check_suite_focus=true)

Maybe be related to https://github.com/rust-lang/rust/issues/87481#issuecomment-887698364?